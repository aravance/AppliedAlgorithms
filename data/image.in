2 2 1 1
XX
XX
1 1 1 1
X
6 7 3 3
XXXXXXX
XXXXXXX
XXXXXXX
XXXXXXX
XXXXXXX
XXXXXXX
20 20 17 16
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
....................
...............X....
....................
....................
....................
20 20 20 20
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
XXXXXXXXXXXXXXXXXXXX
6 4 2 3
.XXX
.XXX
.XXX
...X
..X.
X...
5 6 3 3
.XXXX.
X....X
..XX.X
.X...X
..XXX.
5 6 3 3
.XXX..
X....X
..XX.X
.X...X
...XX.
4 7 1 2
.X.X.X.
XXXXXXX
..X.X.X
XXX.X.X
15 17 8 13
........X........
.......X.X.......
......X...X......
.....X..X..X.....
....X..X....X....
...X..X...X..X...
.....X.XX..X..X..
.X..X..XXX..X..X.
..X..X..X..X..X..
...X......X..X...
....X..X.X..X....
.....X..X..X.....
......X..XX......
.......X.X.......
........X........
15 17 8 10
........X........
.......X.X.......
......X...X......
.....X..X..X.....
....X..X....X....
...X..X...X..X...
.....X.XX..X..X..
.X..X..XXX..X..X.
..X..X..X..X..X..
...X......X..X...
....X..X.X..X....
.....X..X..X.....
......X..XX......
.......X.X.......
........X........
15 17 8 10
........X........
.......X.X.......
......X...X......
.....X..X..X.....
....X..X.X..X....
...X..X...X..X...
.....X.XX..X..X..
.X..X..XXX..X..X.
..X..X..X..X..X..
...X......X..X...
....X..X.X..X....
.....X..X..X.....
......X..XX......
.......X.X.......
........X........
0 0 0 0

