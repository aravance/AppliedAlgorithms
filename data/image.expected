Perimeter of object 1 at (1,1) = 8
Perimeter of object 2 at (1,1) = 4
Perimeter of object 3 at (3,3) = 26
Perimeter of object 4 at (17,16) = 4
Perimeter of object 5 at (20,20) = 80
Perimeter of object 6 at (2,3) = 18
Perimeter of object 7 at (3,3) = 40
Perimeter of object 8 at (3,3) = 10
Perimeter of object 9 at (1,2) = 38
Perimeter of object 10 at (8,13) = 140
Perimeter of object 11 at (8,10) = 36
Perimeter of object 12 at (8,10) = 180

